src/widgets/DataTable.js

```diff
@@ -1,10 +1,10 @@
-import React, {useState, useEffect} from 'react';
+import React, { useState, useEffect } from 'react';
```

Comment:
Not sure why this file is changed.

Reply:
because of my format on save settings - ill change it back